package.json

```diff
@@ -135,7 +135,7 @@
     "ovh-angular-timeline": "^1.5.2",
     "ovh-angular-ui-confirm-modal": "^1.0.2",
     "ovh-angular-user-pref": "^0.3.1",
-    "ovh-api-services": "^8.3.0",
+    "ovh-api-services": "^9.2.0",
```

Comment:
Please also ugprade `@ovh-ux/ng-ovh-api-wrappers`

Reply:
the develop branch wasn't up to date?